src/software/ml/ml_design.md

```diff
@@ -0,0 +1,85 @@
+# MACHINE LEARNING DESIGN
+Overview of  machine learning and training model frame work.
+
+<!-- <div class="warning">
+
+**Section**
+
+- [ 
+-  
+
+</div> -->
+
+## Data set
+
+
+**kaggle.com**
+- https://www.kaggle.com/code/gleblevankov/meow-meow
+- This dataset, composed of 440 sound recordings, contains meows emitted by cats in different contexts. Specifically, 21 cats belonging to 2 breeds (Maine Coon and European Shorthair) have been repeatedly exposed to three different stimuli that act as labels for prediction:
+
+**Cat sound**
+ ![](../../images/ml/es_hungry.png)
+ 
+- Example Sound of Hungry cat.
+ 
+<p style="text-align: center;">Sound wave hungry</p> 
+<p align="center"><img height ="300px" class="center-block" src="../../images/ml/es_hungry.png"></p>
+
+<p style="text-align: center;">  Mel frequency cepstrum coefficient</p>
+<p align="center"><img height ="300px" class="center-block" src="../../images/ml/sp_hungry.png"></p>
+ 
+ 
+
+- Example Sound of Isolated cat.
+
+<p style="text-align: center;">Sound wave Isolated</p> 
+<p align="center"><img height ="300px" class="center-block" src="../../images/ml/es_isolated.png"></p>
+
+<p style="text-align: center;">  Mel frequency cepstrum Isolated</p>
+<p align="center"><img height ="300px" class="center-block" src="../../images/ml/sp_isolated.png"></p>
+ 
+
+- Example Sound of Happy cat.
+<p style="text-align: center;">Sound wave Happy</p> 
+<p align="center"><img height ="300px" class="center-block" src="../../images/ml/es_happy.png"></p>
+
+<p style="text-align: center;">  Mel frequency cepstrum Happy</p>
+<p align="center"><img height ="300px" class="center-block" src="../../images/ml/sp_happy.png"></p>
+ 
+
+
+**Features Extraction form Cat sound**
+- form Sound wave we can Extraction Features with by splitting the audio into short segments to obtain a sequence of frequency spectra.  each covering only a small slice of time, and stack the resulting spectra together into a spectrogram. 
```

Comment:
We can extract features from sound wave by splitting the audio into short segments to obtain a sequence of frequency spectra,  each covering only a small slice of time, and stack the resulting spectra together into a spectrogram.